artiq/firmware/ksupport/nrt_bus.rs

```diff
@@ -43,6 +43,17 @@ pub mod i2c {
             data
         }) as i32
     }
+
+    pub extern fn switch_select(busno: i32, address: i32, channel: i32, clear: bool) {
```

Comment:
DId you compile-test this?
Anyway I think the clear/mask argument should be removed?